modules/stanford_profile_styles/lib/scss/admin/ckeditor.scss

```diff
@@ -0,0 +1,10 @@
+@charset 'UTF-8';
+// This stylesheet creates a css file that is attached to the CKEDITOR WYSIWYG.
+@import "../paragraph-types/wysiwyg";
+
+// This extend will duplicate the su-wysiwyg-text styles in to this stylesheet
+// and add a secondary set of children selectors under the body tag. This is an
+// acceptable amount of bloat for maintaining styles automatically.
+body {
+  @extend .su-wysiwyg-text;
```

Comment:
would this be best suited for `stanford_text_editor`?

Reply:
I'd argue no because different themes and different modules can influence the look and feel and we should not assume D8Core.

Reply:
maybe in the theme? we have ckeditor styles already there https://github.com/SU-SWS/stanford_basic/blob/8.x-4.x/stanford_basic.info.yml#L34

Reply:
These are specific overrides just for when we use a WYSIWYG editor. We don't want to mess with the default out of the box Decanter styles that are already included in that theme's file.